R/plumber-step.R

```diff
@@ -190,6 +191,8 @@ PlumberEndpoint <- R6Class(
       }
       private$envir <- envir
 
+      private$regex <- createPathRegex(path, self$getFuncParams())
```

Comment:
@schloerke to look into removing this function completely and replace with known parsed structure from `parseBlock()`

Reply:
Be aware, this is also relevant to swagger changes

Reply:
This is the parseBlock code path.  The only time the path is parsed is in `createPathRegex()`.

Resolving comment